services.yaml

```diff
@@ -1780,37 +1808,36 @@ definitions:
           $ref: "#/definitions/ViewBasic"
   Filter:
     description: Filter properties
-    type: "object"
+    type: object
     properties:
-      name:
-        type: "string"
-        description: Use a filter name from the list above.
-        enum: [SERVERS, APPS, DEPLOYMENTS, FIRST SEEN CUSTOM, FIRST SEEN PRESET, EVENT TYPE, EVENT NAME, EVENT LOCATION, EVENT PACKAGE, ENTRY POINT, LABELS, INTRODUCED BY, SEARCH]
-      values:
-        type: "array"
-        description: The values that shall be filtered. Follow the examples to define proper value format that matches desired filter type.
+      event_type:
+        type: string
+        enum: [Uncaught Exceptions, Caught Exceptions, Swallowed Exceptions, Logged Warnings, Logged Errors, HTTP Errors, Timers]
+      first_seen:
+        type: object
+        description: Defines the first time an event occured using a absolute time frame or a relative ime frame. If absolute time frame is defined, the relative will be ignored.
+        properties:
+          absolute:
+            type: object
+            properties:
+              from:
+                type: string
+                description: Start time. Date in ISO8601 format (yyyy-MM-ddTHH:mm:ss.SSSZZ)
+                example: 2018-02-28T22:47:09.038+02:00
+              to:
+                type: string
+                description: End time. Date in ISO8601 format (yyyy-MM-ddTHH:mm:ss.SSSZZ)
+                example: 2018-02-28T22:47:09.038+02:00
+          relative:
+            type: string
+            enum: [last_hour, last_3h, last_6h, last_12h, last_24h, last_48h, last_7d, last_14d, last_30d]
+            example: "last_6h"
+      labels:
```

Comment:
what about the rest of the filters?

Reply:
these are the filters which are currently supported. Do we want to document the others?

Reply:
all possible filters were documented. The filters which are not implemented were marked